client/src/pages/Login.js

```diff
@@ -0,0 +1,192 @@
+import React, {Component, isValidElement} from "react";
+
+class Login extends Component {
```

Comment:
Maybe a question to discuss as a group but since React is moving away from class components towards functional components, I think we should refactor this to be a functional component